index.html

```diff
@@ -41,5 +41,142 @@ <h1 class="header-text">Hey there. I’m Raphael
             <div><img src="img/Action.png" alt="scroll"></div>
         </div>
     </header>
+
+    <main>
+        <div class="about-work-title">
+            <div>
+            <h2>My Recent Works</h2>
+            <div class="bar"></div>
+            </div>
+            
+        </div>
+
+        <div class="cards-container">
+            <div class="card-item">
+                <div class="up-card"></div>
+                <div class="down-card">
+                    <p>Multi-Post Stories Gain+Glory</p>
+                    <div class="skills"><div>Ruby on rails</div><div>CSS</div><div>JavScript</div><div>html</div></div>
+                    <div class="button"><p>See Project</p></div>
+                </div>    
+            </div>
+
+
+            <div class="card-item">
+                
+                <div class="down-card">
+                    <p>Multi-Post Stories Gain+Glory</p>
+                    <div class="skills"><div>Ruby on rails</div><div>CSS</div><div>JavScript</div><div>html</div></div>
+                    <div class="button"><p>See Project</p></div>
+                </div>    
+            </div>
+
+
+            <div class="card-item">
+                
+                <div class="down-card">
+                    <p>Multi-Post Stories Gain+Glory</p>
+                    <div class="skills"><div>Ruby on rails</div><div>CSS</div><div>JavScript</div><div>html</div></div>
```

Comment:
To follow HTML semantic guidelines, I suggest using a `ul` tag as a wrapper for the skills and each individual child a `li` tag.